docs/services/slam/cartographer/_index.md

```diff
@@ -34,318 +22,467 @@ The source code for this module is available on the [`viam-cartographer` GitHub
-    {
-      "attributes": {
-        "config_params": {
-          "max_range_meters": "25",
-          "mode": "2d",
-          "min_range_meters": "0.2"
-        },
-        "camera": {
-          "name": "rplidar",
-          "data_frequency_hz": "5"
+      "modules": [
+        {
+          "type": "registry",
+          "name": "viam_rplidar",
+          "module_id": "viam:rplidar",
+          "version": "0.1.14"
         },
-        "enable_mapping": true,
-        "use_cloud_slam": true
-      },
-      "name": "slam",
-      "type": "slam",
-      "namespace": "rdk",
-      "model": "viam:slam:cartographer"
-    },
-    {
-      "name": "Data-Management-Service",
-      "type": "data_manager",
-      "attributes": {
-        "tags": [],
-        "additional_sync_paths": [],
-        "sync_interval_mins": 0.1,
-        "capture_dir": ""
-      }
-    }
-  ],
-  "components": [
-    {
-      "namespace": "rdk",
-      "attributes": {},
-      "depends_on": [],
-      "service_configs": [
         {
+          "type": "registry",
+          "name": "viam_cartographer",
+          "module_id": "viam:cartographer",
+          "version": "0.3.36"
+        }
+      ],
+      "services": [
+        {
+          "type": "slam",
+          "namespace": "rdk",
+          "model": "viam:slam:cartographer",
           "attributes": {
-            "capture_methods": [
-              {
-                "disabled": false,
-                "method": "NextPointCloud",
-                "capture_frequency_hz": 5
-              }
-            ]
+            "config_params": {
+              "min_range_meters": "0.2",
+              "max_range_meters": "25",
+              "mode": "2d"
+            },
+            "camera": {
+              "name": "rplidar",
+              "data_frequency_hz": "5"
+            },
+            "enable_mapping": false,
+            "use_cloud_slam": false,
+            "existing_map": "${packages.slam_map.test-map-1}/internalState.pbstream"
           },
-          "type": "data_manager"
+          "name": "slam"
         }
       ],
-      "name": "rplidar",
-      "model": "viam:lidar:rplidar",
-      "type": "camera"
+      "components": [
+        {
+          "model": "viam:lidar:rplidar",
+          "type": "camera",
+          "namespace": "rdk",
+          "attributes": {},
+          "depends_on": [],
+          "name": "rplidar"
+        }
+      ],
+      "packages": [
+        {
+          "type": "slam_map",
+          "name": "test-map-1",
+          "version": "1697208847",
+          "package": "d1c224e8-483e-4cc7-980f-76b89d8fb507/test-map-1"
+        }
+      ]
     }
-  ]
-}
-```
-
-{{% /expand%}}
+    ```
 
-#### Update existing map
+    {{% /tab %}}
+    {{< /tabs >}}
 
-This mode is similar to [Create new map](#create-new-map), except it creates a new version of an existing map in the robot's location. (It does not overwrite the old version of the map.)
+    {{< alert title="Tip" color="tip" >}}
```

Comment:
The formatting of this tip box seems a little weird:

<img width="820" alt="Screenshot 2023-10-20 at 12 10 36 PM" src="https://github.com/viamrobotics/docs/assets/1517691/35c35be4-09bb-40e0-bc45-c88c2502e03d">